_posts/0000-01-02-Motya-cobol.md

```diff
@@ -0,0 +1,6 @@
+--
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#This is my first change to a file
## So far it's not too bad
[Here is a link to my website](https://annexosteopathy.com)